<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="styleSheet">
      <string notr="true">QPushButton{
 border:1px solid green;
border-radius: 15px;
background:rgb(158, 255, 106)
}
QPushButton:hover {
  background-color: #3e8e41;
  box-shadow: 0 5px #666;
  transform: translateY(4px);
}
QPushButton:pressed {
  background-color:rgb(158, 255, 106) ;
  
}
QLineEdit{
border-radius: 15px;
}</string>
     </property>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-10</y>
        <width>800</width>
        <height>561</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">#frame{
background:url(:/test/bg_menu.png)
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>260</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Username</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>300</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QPushButton" name="loginBtn">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>260</y>
         <width>71</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
      <widget class="QPushButton" name="signupBtn_3">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>350</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#signupBtn_3{
 border-radius: 15px;
}</string>
       </property>
       <property name="text">
        <string>Đăng kí</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>150</y>
         <width>501</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#label_9{
image:url(:/test/oanqan.png)
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <property name="styleSheet">
      <string notr="true">QPushButton{
 border:2px solid black;
border-radius: 15px;
background:rgb(158, 255, 106)
}

QPushButton:hover {
  background-color: #3e8e41;
  box-shadow: 0 5px #666;
  transform: translateY(4px);
}
QPushButton:pressed {
  background-color:rgb(158, 255, 106) ;
  
}</string>
     </property>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>571</width>
        <height>301</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#listWidget{
background:white
}
QScrollBar:vertical {
border: 2px solid grey;
}
QScrollBar:vertical {
    border: 2px solid grey;
    background: rgb(158, 255, 106)


}
QScrollBar::handle:vertical {
    background: white;
    min-height: 20px;
}
QScrollBar::add-line:vertical {
    border: 2px solid grey;
    background: rgb(158, 255, 106);
    height: 20px;
}

QScrollBar::sub-line:vertical {
    border: 2px solid grey;
    background: rgb(158, 255, 106);
    height: 20px;

}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nameSortBtn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>411</width>
        <height>31</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 border:2px solid black;
border-radius: 15px;
}</string>
      </property>
      <property name="text">
       <string>Tên</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rankSortBtn">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>0</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Điểm xếp hạng</string>
      </property>
     </widget>
     <widget class="QLabel" name="ava">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>0</y>
        <width>231</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#ava{
image:url(:/ava/ava2.jpg);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_name">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>230</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#label_name{
 border:3px outset rgb(157, 255, 146);
background:rgb(24, 190, 49);
color:black}</string>
      </property>
      <property name="text">
       <string>Test_User123</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_rank">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>270</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#label_rank{
 border:3px outset rgb(157, 255, 146);
background:rgb(24, 190, 49)
}</string>
      </property>
      <property name="text">
       <string>Rank</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
